samples/openthread/coprocessor/README.rst

```diff
@@ -136,6 +136,16 @@ Building and running
 
 .. include:: /includes/build_and_run.txt
 
+Minimal RCP build
```

Comment:
I would move this content to the previous _Configuration files_ section. No need to explain how to use the overlay file here.

Reply:
Yes, exactly, that's what this section is for. @lmaciejonczyk , let me know if you want me to add a commit.

And btw, that Configuration section is actually inconsistent with the other Thread samples, but they'll be updated at some point as well. :)

Reply:
@ru-fu do all required changes here please